23:10:03.275 [main] DEBUG org.apache.ibatis.logging.LogFactory - Logging initialized using 'class org.apache.ibatis.logging.slf4j.Slf4jImpl' adapter.
23:10:03.650 [main] DEBUG o.a.i.d.pooled.PooledDataSource - PooledDataSource forcefully closed/removed all connections.
23:10:03.650 [main] DEBUG o.a.i.d.pooled.PooledDataSource - PooledDataSource forcefully closed/removed all connections.
23:10:03.650 [main] DEBUG o.a.i.d.pooled.PooledDataSource - PooledDataSource forcefully closed/removed all connections.
23:10:03.650 [main] DEBUG o.a.i.d.pooled.PooledDataSource - PooledDataSource forcefully closed/removed all connections.
23:10:03.650 [main] DEBUG o.a.i.d.pooled.PooledDataSource - PooledDataSource forcefully closed/removed all connections.
23:10:03.650 [main] DEBUG o.a.i.d.pooled.PooledDataSource - PooledDataSource forcefully closed/removed all connections.
23:10:04.586 [main] DEBUG o.a.i.d.pooled.PooledDataSource - Created connection 32880125.
23:10:04.586 [main] DEBUG o.a.i.d.pooled.PooledDataSource - Testing connection 32880125 ...
23:10:04.632 [main] WARN  o.a.i.d.pooled.PooledDataSource - Execution of ping query 'NO PING QUERY SET' failed: Erreur de syntaxe près de 'NO PING QUE
RY SET' à la ligne 1
23:10:04.632 [main] DEBUG o.a.i.d.pooled.PooledDataSource - Connection 32880125 is BAD: Erreur de syntaxe près de 'NO PING QUERY SET' à la ligne 1
23:10:04.632 [main] DEBUG o.a.i.d.pooled.PooledDataSource - A bad connection (32880125) was returned from the pool, getting another connection.
23:10:04.648 [main] DEBUG o.a.i.d.pooled.PooledDataSource - Created connection 17196824.
23:10:04.648 [main] DEBUG o.a.i.d.pooled.PooledDataSource - Returned connection 17196824 to pool.
23:10:04.925 [main] DEBUG o.a.i.t.jdbc.JdbcTransaction - Opening JDBC Connection
23:10:04.925 [main] DEBUG o.a.i.d.pooled.PooledDataSource - Checked out connection 17196824 from pool.
23:10:04.925 [main] DEBUG o.a.i.d.pooled.PooledDataSource - Testing connection 17196824 ...
23:10:04.925 [main] WARN  o.a.i.d.pooled.PooledDataSource - Execution of ping query 'NO PING QUERY SET' failed: Erreur de syntaxe près de 'NO PING QUE
RY SET' à la ligne 1
23:10:04.925 [main] DEBUG o.a.i.d.pooled.PooledDataSource - Connection 17196824 is BAD: Erreur de syntaxe près de 'NO PING QUERY SET' à la ligne 1
23:10:04.925 [main] DEBUG o.a.i.d.pooled.PooledDataSource - A bad connection (17196824) was returned from the pool, getting another connection.
23:10:04.941 [main] DEBUG o.a.i.d.pooled.PooledDataSource - Created connection 5403403.
23:10:04.941 [main] DEBUG o.a.i.t.jdbc.JdbcTransaction - Setting autocommit to false on JDBC Connection [com.mysql.jdbc.JDBC4Connection@52730b]
23:10:04.956 [main] DEBUG i.s.m.s.mapper.BlogMapper.selectBlog - ==>  Preparing: select * from blog where id = ? 
23:10:05.065 [main] DEBUG i.s.m.s.mapper.BlogMapper.selectBlog - ==> Parameters: 100(String)
23:10:05.175 [main] DEBUG i.s.m.s.mapper.BlogMapper.selectBlog - <==      Total: 1
23:10:05.175 [main] DEBUG o.a.i.t.jdbc.JdbcTransaction - Resetting autocommit to true on JDBC Connection [com.mysql.jdbc.JDBC4Connection@52730b]
23:10:05.175 [main] DEBUG o.a.i.t.jdbc.JdbcTransaction - Closing JDBC Connection [com.mysql.jdbc.JDBC4Connection@52730b]
23:10:05.190 [main] DEBUG o.a.i.d.pooled.PooledDataSource - Testing connection 5403403 ...
23:10:05.190 [main] WARN  o.a.i.d.pooled.PooledDataSource - Execution of ping query 'NO PING QUERY SET' failed: Erreur de syntaxe près de 'NO PING QUE
RY SET' à la ligne 1
23:10:05.190 [main] DEBUG o.a.i.d.pooled.PooledDataSource - Connection 5403403 is BAD: Erreur de syntaxe près de 'NO PING QUERY SET' à la ligne 1
23:10:05.190 [main] DEBUG o.a.i.d.pooled.PooledDataSource - A bad connection (5403403) attempted to return to the pool, discarding connection.
23:10:05.190 [main] DEBUG i.s.m.s.service.impl.BlogServiceTest - title:标题1
